workloads/server.hbs.md

```diff
@@ -105,3 +105,285 @@ The following values are valid within the `ports` argument:
 | `port` | The port on which the application is exposed to the rest of the cluster |
 | `containerPort` | The port on which the application listens for requests. Defaults to `port` if not set. |
 | `name` | A human-readable name for the port. Defaults to `port` if not set. |
+
+
+## <a id="params"></a> Securing `server` workloads
+
+### Manual configuration for HTTP workloads
+For http `server` workloads you can expose them by creating an Ingress resource and using cert-manager to provision TLS signed certificates. Taking the `spring-sensors-consumer-web`
+workload as an example, create the following `Ingress`:
+
+```
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: spring-sensors-consumer-web
+  namespace: <YOUR-DEVELOPER-NAMESPACE>
+  annotations:
+    cert-manager.io/cluster-issuer: tap-ingress-selfsigned
+    ingress.kubernetes.io/force-ssl-redirect: "true"
+    kubernetes.io/ingress.class: contour
+    kubernetes.io/tls-acme: "true"
+spec:
+  tls:
+    - secretName: spring-sensors-consumer-web
+      hosts:
+        - "spring-sensors-consumer-web.<INGRESS-DOMAIN>"
+  rules:
+    - host: "spring-sensors-consumer-web.<INGRESS-DOMAIN>"
+      http:
+        paths:
+          - pathType: Prefix
+            path: /
+            backend:
+              service:
+                name: spring-sensors-consumer-web
+                port:
+                  number: 8080
+
+```
+
+Replace the `<YOUR-DEVELOPER-NAMESPACE>` with your developer namespace and `<INGRESS-DOMAIN>` with the domain name defined
+in `tap-values.yaml` during the installation. Also, set the annotation `cert-manager.io/cluster-issuer` to the `shared.ingress_issuer` value
+configured during installation or leave it as `tap-ingress-selfsigned` to use the default one. Make sure to update the port exposed by
+your `Service` resource, in the previous snippet it is set to `8080`. 
+
+Your `server` workload can be accessed with https like so:
+```console
+curl -k https://spring-sensors-consumer-web.<INGRESS-DOMAIN>
+```
+
+
+### Creating a new workload type that will secure _server_ workloads by default
```

Comment:
The other thing I was thinking... Other than the manual configuration above, do we want to provide public docs on how to setup this by default? Should we confirm with Shaine or Evan? This is creating another workload type, which is cool, but I was wondering if it is a desirable info to make it available in the public docs.

Reply:
I will point them to this documentation and ask them about this.

Reply:
I think it's useful to include the notion in the documentation that you can create additional workloads, and a sample of how to do so.

Reply:
We might even want to promote this to "defining your own workload type", but let's start here, since the other would require a bunch more authoring work.

Reply:
I changed this title to be: "Define your own workload type that will expose _server_ workloads outside the cluster"

to be consistent with the previous "Exposing `server` workloads outside the cluster"